src/main/java/io/github/cadenceoss/iwf/core/UntypedClient.java

```
@@ -50,7 +53,16 @@ public String StartWorkflow(
                 .iwfWorkflowType(workflowType)
                 .workflowTimeoutSeconds(options.getWorkflowTimeoutSeconds())
                 .stateInput(clientOptions.getObjectEncoder().encode(input))
-                .startStateId(startStateId));
+                .startStateId(startStateId)
+                .workflowStartOptions(
+                        new io.github.cadenceoss.iwf.gen.models.WorkflowStartOptions()
+                                .workflowIDReusePolicy(WorkflowIdReusePolicyMapper.toGenerated(
+                                        options.getWorkflowIdReusePolicy()))
+                                .cronSchedule(
+                                        CronScheduleValidator.isValidCronSchedule(options.getCronSchedule())
+                                                ? options.getCronSchedule() : null)
```

I think we should throw error if cron is invalid

---

👍 fixed, it will throw illegal argument exception if it's not valid